tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/restgenerateurl/GenerateUrl.java

```diff
@@ -0,0 +1,17 @@
+package ca.bc.gov.open.jagefilingapi.qa.restgenerateurl;
+
+import ca.bc.gov.open.jagefilingapi.qa.restdata.Payload;
+import static io.restassured.RestAssured.*;
+import static org.hamcrest.Matchers.*;
+
+
+public class GenerateUrl {
+
+   public void generateEfilingUrl() {
+        baseURI = "http://localhost:8080";
```

Comment:
can this be env var instead of hard coded url val?

Reply:
URI value is passed from the properties file now.